pangram/pangram.py

```diff
@@ -1,2 +1,9 @@
 def is_pangram(sentence):
-    pass
+    
+    # ensure string is lowercase
+    sentence = sentence.lower()
+    
+    # remove some punctuation
+    for thing in ' ,.!?:;()':
```

Comment:
Add the $ symbol? ex. "The price of the stock drop to $1.50 per share"
Would that take into account also the newline?

Reply:
Got it. Added $ in most recent commit. Also added a docstring.